### PR TITLE
Switch to Central Package Management and update binding redirect

### PR DESCRIPTION
**Pull request by ivandrofly**
~Replaced `packages.config` with `PackageReference` in `Test.csproj` for streamlined package management. Also updated the `System.Net.Http` binding redirect in `app.config` from version `4.2.0.0` to `4.1.2.0`.~

~### [Benefits of using package references (Click here)](https://learn.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference#benefits-of-using-packagereference)~

Update to central package management

### CODE REVIEW

File: src/Test/app.config [hunk 1]
```diff
@@ -24,7 +24,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.2.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0" />
```
Comment (ivandrofly) [inline src/Test/app.config:27]: This out automatically done, but after checking also on the nuget, there 
seems to be only 4.1.2 available and not 4.2.0

**Comment by ivandrofly** [comment]
Question here (image bellow)
image


Is this section really needed? It cause build issue with Rider IDE.

If you delete the `obj` folder and try to build/restore - Rider will start failing to build and complaining about the missing
dependencies and I was only able to build & restore package after commenting out that section.

File: src/Test/Test.csproj [hunk 2]
```diff
@@ -1,224 +1,223 @@
...
+        <AssemblyName>Test</AssemblyName>
+        <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+        <FileAlignment>512</FileAlignment>
+        <OutputType>Library</OutputType>
+        <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+        <ProjectGuid>{7BE5B8E8-9469-4C7C-89D7-E8C884DEFC0E}</ProjectGuid>
+        <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+        <RootNamespace>Test</RootNamespace>
+        <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+        <RuntimeIdentifier>win</RuntimeIdentifier>
```
Comment (ivandrofly) [inline src/Test/Test.csproj:15]: Added 

See: https://ci.appveyor.com/project/SubtitleEdit/subtitleedit/builds/50515390

File: src/Test/Test.csproj [hunk 3]
```diff
@@ -1,224 +1,223 @@
...
-    <FileAlignment>512</FileAlignment>
-    <OutputType>Library</OutputType>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{7BE5B8E8-9469-4C7C-89D7-E8C884DEFC0E}</ProjectGuid>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <RootNamespace>Test</RootNamespace>
-    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
-    <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
```
Comment (ivandrofly) [inline src/Test/Test.csproj:17]: Removed elements 

- TargetFrameworkProfile
- NuGetPackageImportStamp

**Comment by niksedk** [comment]
What about Central Package Management?
https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management

**Comment by ivandrofly** [comment]
> What about Central Package Management?
> 
> https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management

Interesting. Ill take a look

**Comment by ivandrofly** [comment]
@niksedk  maybe you need to configure something appveyor?

https://ci.appveyor.com/project/SubtitleEdit/subtitleedit/builds/50527318

**Comment by ivandrofly** [comment]
> @niksedk maybe you need to configure something appveyor?
> 
> https://ci.appveyor.com/project/SubtitleEdit/subtitleedit/builds/50527318

Never mind, it's all good now

**Pull request merged**